history/history-network.md

```diff
@@ -19,7 +19,7 @@ In addition, the chain history network provides individual epoch accumulators fo
     * Transactions
     * Ommers
 * Receipts
-* Header epoch accumulators (pre-merge only)
+* Header epoch records (pre-merge only)
```

Comment:
I'm not so sure about the new `EpochRecord`s naming.

Reply:
Seems fine to me.